app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -541,5 +562,9 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         const val EXTRA_OPENED_FROM_NOTIFICATION = "notification_open"
         const val delay = 5000L
         const val START_IN_RECENTS_SCREEN = "start_in_recents_screen"
+
+        // PWA must have been used within last 30 days to be considered "recently used" for the
+        // telemetry purposes.
+        const val PWA_RECENTLY_USED_THRESHOLD = 1000 * 60 * 60 * 24 * 30L
```

Comment:
I would prefer vals to be used here instead of magic numbers

Reply:
Switched to `DateUtils`